drivers/include/motor_driver.h

```diff
@@ -105,29 +106,32 @@ extern "C" {
 /** @} */
 
 /**
- * @brief Macro to return motor driver id
+ * @brief Describe DC motor driver modes
  */
-#define MOTOR_DRIVER_DEV(x) (x)
+typedef enum {
+    MOTOR_DRIVER_2_DIRS         = 0,            /**< 2 GPIOs for direction, \
+                                                     handling brake */
+    MOTOR_DRIVER_1_DIR          = 1,            /**< single GPIO for direction, \
+                                                     no brake */
+    MOTOR_DRIVER_1_DIR_BRAKE    = 2             /**< single GPIO for direction, \
+                                                     single GPIO for brake */
+} motor_driver_mode_t;
 
 /**
- * @brief Describe DC motor driver modes
+ * @brief Describe DC motor driver brake levels
```

Comment:
? this is about braking with more or less force / acceleration ?

Reply:
done